tests/cases/conformance/jsx/tsxSpreadAttributesResolution2.tsx

```diff
@@ -27,4 +27,5 @@ let p = <Poisoned {...obj} />;
 let y = <Poisoned />;
 let z = <Poisoned x y/>;
 let w = <Poisoned {...{x: 5, y: "2"}}/>;
-let w1 = <Poisoned {...{x: 5, y: "2"}} X="hi" />;
\ No newline at end of file
+let w1 = <Poisoned {...{x: 5, y: "2"}} X="hi" />;
```

Comment:
This also had an incorrect `x` in the spread so after the change a typo was not suggested for `X`. I added an extra test case to keep the typo suggestion as a test case.